### ISSUE TITLE
Simplify + Standardize OS Buttons

### ISSUE THREAD

**Issue by ninavizz** [QubesOS/qubes-issues#6167]
**Problem**
Today's buttons in Qubes OS go against a few standard paradigms in UX.
- There should always be a darker "primary" and a lighter "secondary" button
- The "back-arrow" icon is unclear to many users as a return-button. My Lenovo laptop and my Mac, both lack this symbol on any buttons, as an example.
- Icons on buttons as a general rule, are undesirable. They add clutter to a UI which adds cognitive load. While they are often seen as helpful in critical review, in reality there are often too many icons in a user's view; which when the icons are not clearly and obviously evident, and when many have gradients and multiple colors, adds to a user's cognitive burden.
- In a discussion on a different thread from many months ago that I cannot remember, the "x" on the cancel button was expressed by many as desirable; so I included a simpler "x" on the below mockups.

**Proposed Solution**
The below draws from the Gnome UI library. The mockups below use SegoeUI, but I'd also like to standardize around Source Sans Pro w/in Qubes, eventually; so that'd be preferred.

![ButtonStyles](https://user-images.githubusercontent.com/8262612/97765227-8ad91600-1ace-11eb-9c1c-737513a609e5.png)

**Comment by GWeck** [comment]
The buttons should also be presented in a defined sequence. Currently, sometimes the OK button is on the left side and the Cancel button on the right, somtimes vice versa. Surely this helps to alert the user, but it can also trigger to select the wrong button.

**Comment by andrewdavidwong** [comment]
> Today's buttons in Qubes OS go against a few standard best-practice* paradigms in UX.

I'm going to assume that this is referring to Qubes-specific GUIs (manager/widgets), since non-Qubes-specific (e.g., Xfce4) GUIs are inherited from upstream.

**Comment by ninavizz** [comment]
Oof... I was hoping this could even go on XFCE managed windows, as those are the default for Qubes.

**Comment by ninavizz** [comment]
Artwork for the "x" on an aside:
![Exxy-X_L](https://user-images.githubusercontent.com/8262612/97794072-11b2ef00-1bb2-11eb-9994-1c06839e90f4.png)
![Exxy-X_M](https://user-images.githubusercontent.com/8262612/97794073-15df0c80-1bb2-11eb-822a-22de8a4f038d.png)
![Exxy-X_S](https://user-images.githubusercontent.com/8262612/97794074-18d9fd00-1bb2-11eb-99bc-5eaa60d6fe1d.png)

**Comment by marmarek** [comment]
It is possible to create our own theme for Xfce. I don't know how much work that is and how much flexibility it gives. Take a look at "Appearance" option in settings - there are "Style" and "Icons" tabs. I don't see any of them having different colors for different buttons, so I don't know if that's possible, but there is something to explore.

**Comment by andrewdavidwong** [comment]
> It is possible to create our own theme for Xfce. I don't know how much work that is and how much flexibility it gives. Take a look at "Appearance" option in settings - there are "Style" and "Icons" tabs. I don't see any of them having different colors for different buttons, so I don't know if that's possible, but there is something to explore.

Ah, that's a good point. A long time ago, I customized an Xfce theme for myself, and I vaguely recall seeing a way to customize buttons, but I had (and still have) no idea how to do it. My sense is that the overall framework allows for quite a bit of flexibility, if you know how to use it.

**Comment by ninavizz** [comment]
Right. I'd like to see Qubes ship with cleaner & more intuitive defaults. I think I discussed this with @marmarta some time ago, and she suggested it could be easy. Marta, could you summarize what you'd had in mind, here?

Of course, world peace could be easy, too... if a few basic caveats could be met by all of humanity. Pride, greed, egos, etc. Small things, really.

**Comment by ninavizz** [comment]
Adding this as a component to #6414

**Comment by ninavizz** [comment]
Labeled this issue "Help Wanted," because none of the core Qubes OS team are likely to be able to prioritize this anytime soon, abreast critical project priorities. That said: to me, this feels like an helpful addition to the UI. 

The core Qubes team have given their blessing to this UX solution, so should a developer like to take this on pls "@" me here and I'll hop-to gathering final SVG artwork to make this possible.

**Comment by ben-grande** [comment]
@marmarta Can you check if it is missing something?

> Today's buttons in Qubes OS go against a few standard best-practice* paradigms in UX.
> 
>     * There should always be a darker "primary" and a lighter "secondary" button

Has:

- Create new qube
- Qubes Global Config
- Policy Editor

Has not:

- Qube manager (has OK selected, a bit darker)
- VM Settings
- Backup

>     * The "back-arrow" icon is unclear to many users as a return-button. My Lenovo laptop and my Mac, both lack this symbol on any buttons, as an example.

Need to find where this is used. I see it on Qube Settings "OK" button.

>     * Icons on buttons as a general rule, are undesirable. They add clutter to a UI which adds cognitive load. While they are often seen as helpful in critical review, in reality there are often too many icons in a user's view; which when the icons are not clearly and obviously evident, and when many have gradients and multiple colors, adds to a user's cognitive burden.

I don't understand this part, as the image below this comment has icon on buttons.

>     * In a discussion on a different thread from many months ago that I cannot remember, the "x" on the cancel button was expressed by many as desirable; so I included a simpler "x" on the below mockups.

**Comment by marmarta** [comment]
In general, implementing Better Buttons in Qt has been more annoying due to the way styling works. The list of missing things looks correct, and also: I disagree with Nina on the "never do icons on buttons", so there's that :)
